docs/using-hive.md

```diff
@@ -77,87 +234,77 @@ metadata:
 spec:
   baseDomain: hive.example.com
   clusterName: mycluster
-  compute:
-  - name: worker
-    platform:
-      aws:
-        rootVolume:
-          iops: 100
-          size: 22
-          type: gp2
-        type: m4.large
-    replicas: 3
-  controlPlane:
-    name: master
-    platform:
-      aws:
-        rootVolume:
-          iops: 100
-          size: 22
-          type: gp2
-        type: m4.large
-    replicas: 3
-  imageSet:
-    name: openshift-v4.2.0
-  images:
-    installerImagePullPolicy: Always
-  networking:
-    clusterNetworks:
-    - cidr: 10.128.0.0/14
-      hostSubnetLength: 23
-    machineCIDR: 10.0.0.0/16
-    serviceCIDR: 172.30.0.0/16
-    type: OpenShiftSDN
+  controlPlaneConfig:
+    servingCertificates: {}
+  installed: false
   platform:
     aws:
-      region: us-east-1
-  platformSecrets:
-    aws:
-      credentials:
+      credentialsSecretRef:
         name: mycluster-aws-creds
-  pullSecret:
+      region: us-east-1
+  provisioning:
+    imageSetRef:
+      name: openshift-v4.3.0
+    installConfigSecretRef:
+      name: mycluster-install-config
+    sshPrivateKeySecretRef:
+      name: mycluster-ssh-key
+  pullSecretRef:
     name: mycluster-pull-secret
-  sshKey:
-    name: mycluster-ssh-key
 ```
 
-The hiveutil CLI (see `make hiveutil`) offers a create-cluster command for generating a cluster deployment and submitting it to the Hive cluster using your current kubeconfig.
-
-To view what create-cluster generates, *without* submitting it to the API server, add `-o yaml` to the above command. If you need to make any changes not supported by create-cluster options, the output can be saved, edited, and then submitted with `oc apply`.
+For Azure, replace `spec.platform` with:
```

Comment:
This is not clear. It should be "replace the contents of `spec.platform` with".

Reply:
done